package.json

```diff
@@ -132,8 +138,10 @@
     "babel-plugin-transform-inline-environment-variables": "0.4.3",
     "chalk": "2.4.2",
     "cheerio": "1.0.0-rc.3",
+    "cypress": "3.4.1",
     "eslint": "6.1.0",
     "eslint-config-standard": "13.0.1",
+    "eslint-plugin-cypress": "2.6.1",
```

Comment:
What is this for? If it's possible to avoid adding a dependency (as I suspect in this case there is), it shouldn't be added.

If you're using this for the `describe` function, then you can add a standard eslint comment-option at the top of each test file to declare it as a global.

Reply:
No, this is much more than `describe()`. It also warns you about some [Cypress Best Practices](https://github.com/cypress-io/eslint-plugin-cypress). If you really want, I can remove it, but I don't think that's a wise move for us that are learning Cypress.